### ISSUE TITLE
Update accessibility documentation

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-org#2309]
We should update our a11y guide further to give more detailed information. 

It's also worth discussing what kind of updates can or should be made at the component level. Core has an a11y section, should react also have this section? Should we pull this information forward?
